articles/connections/passwordless/index.md

```diff
@@ -46,7 +46,6 @@ To implement passwordless login, code your app to follow these steps:
 POST https://${account.namespace}/api/v2/users/
 Authorization: Bearer {Auth0 APIv2 Token}
 Content-Type: 'application/json'
-
 {
```

Comment:
I think the empty line in the code block is causing the parser to close the code block prematurely. This change shouldnt affect the code sample's meaning or usefulness.